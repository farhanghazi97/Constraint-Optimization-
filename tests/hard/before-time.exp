m1:mon 9am
m2:mon 9am
m3:mon 9am
m4:mon 9am
cost:0
