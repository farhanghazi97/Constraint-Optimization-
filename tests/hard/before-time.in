meeting, m1
meeting, m2
meeting, m3
meeting, m4
domain, m1, before 12pm, hard
domain, m2, before 12pm, hard
domain, m3, before 12pm, hard
domain, m4, before 12pm, hard
