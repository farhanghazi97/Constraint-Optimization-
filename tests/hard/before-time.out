m1:fri 10am
m2:fri 10am
m3:fri 10am
m4:fri 10am
cost:0
